config/routes.rb

```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  resources :dishes, only: [:show, :update]
+  resources :chefs, only: :show 
+  namespace :chef do
```

Comment:
It's not required in the user story, but if we wanted this path to include the chef ID, you could do the following and force Rails to use your namespaced controller:
```
resources :chefs, only: :show do
      resources :ingredients, only: :index, controller: "chefs/ingredients"
```